score.go

```diff
@@ -141,8 +138,7 @@ func (d *indexData) scoreFileUsingBM25(fileMatch *FileMatch, doc uint32, cands [
 	// Compute the file length ratio. Usually the calculation would be based on terms, but using
 	// bytes should work fine, as we're just computing a ratio.
 	fileLength := float64(d.boundaries[doc+1] - d.boundaries[doc])
-	numFiles := len(d.boundaries)
```

Comment:
`d.boundaries` has `numFiles+1` entries, the last item being the end of the last file.